scripts/docker-compose-files.txt

```diff
@@ -2,3 +2,4 @@ docker-compose-common.yml
 docker-compose-openmrs.yml
 docker-compose-openmrs-override.yml
 docker-compose-openfn.yml
+docker-compose-backup.yml
```

Comment:
This means we shall also be running backup on our local which is not so helpful in local environments. 
You can just delete this line to disable this in local environment.